src/Symfony/Component/Validator/Resources/translations/validators.fi.xlf

```diff
@@ -366,6 +366,26 @@
                 <source>This value should be between {{ min }} and {{ max }}.</source>
                 <target>Arvon tulisi olla välillä {{ min }} - {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="95">
+                <source>This value is not a valid hostname.</source>
+                <target>Arvo ei ole kelvollinen laitenimi (hostname).</target>
+            </trans-unit>
+            <trans-unit id="96">
+                <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
+                <target>Kokoelmassa olevien elementtien määrän pitää olla monikerta luvulle {{ compared_value }}.</target>
+            </trans-unit>
+            <trans-unit id="97">
+                <source>This value should satisfy at least one of the following constraints:</source>
+                <target>Tämän arvon tulee täyttää vähintään yksi seuraavista rajoitteista:</target>
```

Comment:
Maybe "tarkistuksista:" is better term here?

Reply:
Changed